src/services/status/status.js

```diff
@@ -13,7 +37,7 @@
 
 var DiyaSelector = require('../../DiyaSelector').DiyaSelector;
 var util = require('util');
-
+const Promise = require('bluebird');
```

Comment:
no no, you SHOULD use promise as much as possible. It's just that browserify will bundle bluebird directly into diya-sdk.min.js, which means, if one is to use bluebird for another library, it will need to be imported again, resulting in more network and memory overhead. You should look at how I integrated the Q promise library in diya-sdk, and reproduce de same pattern for bluebird (the ultimate goal being to completely remove Q and only rely on bluebird)